src/js/info.js

```diff
@@ -0,0 +1,38 @@
+$(document).ready(function() {
+
+	var info = window.info = {}
+
+	info.taxonomyInfo = function(label){
+		switch (label) {
+		/* root node */
+		case 'serp':
+			return ['Click the tables on the left to get more information about the facets.',0]
+		case 'intervention':
+			return ['Characteristics of possible solutions.',0]
+		case 'scope':	
+			return ['Scope of an effect or an effect target.',0]
+		case 'planning':
+		case 'design':
+		case 'execution':
+		case 'analysis':
+			return ['Scope lorum ipsum', 'scope']
+		case 'context':
+			return ['Factors in the context affecting the applicability of an intervention.',0]
+		case 'other':
+		case 'sut':
+		case 'information':
+		case 'people':
+			return ['context Laurence Ipsum', 'context']
+		case 'effect':
+			return ['Effect targets, or measured effects of an intervention.',0]
+		case 'adapting':
+		case 'solving':
+		case 'assessing':
+		case 'improving':
+			return ['effect lorum Ipsum', 'effect']
+		
+		default:
+			return 
```

Comment:
Returning this by default allows extended taxonomies to be inspected:
```js
return ['empty', label]
```

But I would recommend you to use an object instead:
```js
return { 
    title: label, 
    description: "Space pirates took your description." 
}
```

Reply:
As well as skipping the switch:
```js
var info = {
    'serp': {
        description: 'Click on stuff to the left please'
    },
    intervention: {
        description: "How stuff gets fixed"
    }
]
function (label) {
    var data = info[label]
    if (data) return data
    else return { description: "Empty Empty Oh So Empty" }
}
```

and maybe use the taxonomy node object as input instead of label.